src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/mapper/AdresseHjelper.kt

```diff
@@ -36,8 +32,8 @@ object AdresseHjelper {
             bostedsadresseBarn.vegadresse == bostedsadresseForelder.vegadresse
     }
 
-    private fun harDeltBosted(barn: BarnMedIdent): Boolean {
-        return barn.deltBosted.any {
+    fun harDeltBosted(barn: BarnMedIdent): Boolean {
+        return barn.deltBosted.filter { !it.metadata.historisk }.any {
             it.startdatoForKontrakt.isBefore(LocalDate.now()) &&
                 (it.sluttdatoForKontrakt == null || it.sluttdatoForKontrakt.isAfter(LocalDate.now()))
```

Comment:
Burde vi kanskje ha en hjelpemetode for `gjeldende()` ? Då vi gjør filtreringen av historisk på flere steder?

Ser nå også at vi kanskje burde ha lite annen sjekk på disse datoene
```suggestion
        return barn.deltBosted.gjeldende().any {
            it.startdatoForKontrakt <= LocalDate.now() &&
                (it.sluttdatoForKontrakt == null || it.sluttdatoForKontrakt >= LocalDate.now())
```

Reply:
Hvis det kun finnes en gjeldende på deltBosted så bytt ut `any` med `let`?